### DISCUSSION TITLE
Problem - Datatable with two checkbox in each row

### DISCUSSION THREAD

**Discussion by diegodasilvaesilva** [primefaces/community#2329] [PrimeReact] [answered]
I'm testing the template sakai-react with datatable. 
This datatable has a column to select checkbox "<Column selectionMode="multiple" exportable={false}></Column>", 
But the datatable shows me two checkboxes in each row

![2024-07-17_06-55](https://github.com/user-attachments/assets/5e55c66c-9abb-4765-83ba-31b491faf5b0)

My code:

"<DataTable
                        ref={dt}
                        value={inspectList}
                        selection={selectedInspects}
                        onSelectionChange={(e) => setSelectedInspects(e.value as any)}
                        dataKey="id"
                        className="datatable-responsive"
                        globalFilter={globalFilter}
                        emptyMessage="not found."
                        header={header}
                        responsiveLayout="scroll"
                        globalFilterFields={['sku.id', 'sku.name']}
                        scrollable
                        scrollHeight="400px"
                    >
                        <Column selectionMode="multiple" exportable={false}></Column>
                        <Column field="id" filter header="id" sortable body={codeBodyTemplate} headerStyle={{ minWidth: '15rem' }}></Column>
                        <Column field="name" filter header="Name" sortable body={nameBodyTemplate} headerStyle={{ minWidth: '15rem' }}></Column>
                        <Column field="inspectStatus" header="Status" body={statusBodyTemplate} sortable headerStyle={{ minWidth: '10rem' }}></Column>
                        <Column body={actionBodyTemplate} headerStyle={{ minWidth: '10rem' }}></Column>
                    </DataTable>
"

Where is the wrong?

My version is:

> sakai-react@10.1.0 dev
> Next.js 14.2.4
"dependencies": {
    "@fortawesome/fontawesome-free": "^6.5.2",
    "@types/node": "20.3.1",
    "@types/react": "18.2.12",
    "@types/react-dom": "18.2.5",
    "chart.js": "4.2.1",
    "mathjs": "^13.0.0",
    "next": "^14.2.4",
    "primeflex": "^3.3.1",
    "primeicons": "^6.0.1",
    "primereact": "^10.6.6",
    "react": "18.2.0",
    "react-dom": "18.2.0",
    "typescript": "5.1.3"
  },

**Comment by melloware** [comment, marked as answer]
You need to update your theme to the latest SASS code.  This has been asked many times just search. Your CSS needs to be updated.
